src/Components/ProjectTile.js

```diff
@@ -0,0 +1,71 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown, faChevronUp } from '@fortawesome/free-solid-svg-icons';
+import { useHistory } from "react-router-dom";
+import Button from "@mui/material/Button";
+import Service from '../Service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8494981)
```suggestion
import Service from "../Service";
```